### ISSUE TITLE
actkbd doesn't launch when plugging in a keyboard on Brewmaster

### ISSUE THREAD

**Issue by sharkwouter** [ValveSoftware/SteamOS#339]
On Brewmaster the udev rule set to launch actkbd when a keyboard is plugged in (for shortcut keys like for returning to Steam and for the bug reporter) doesn't seem to work. Udev does seem to run. On Alchemist this does work.

The configuration file is /etc/udev/rules.d/98-actkbd.rules found in the steamos-base-files package. If you're going to make a change to that package, please also remove the following files from it as they do nothing:

/etc/skel/.config/openbox
/etc/skel/.config/xfce4
/etc/skel/.config/xfce4/xfconf
/etc/skel/.config/xfce4/xfconf/xfce-perchannel-xml

**Comment by johnv-valve** [comment]
actkbd will need some reworking before it will work correctly in the new systemd world. We are not using the bug reporter anymore so we don't have a lot of incentive to do this work.

**Comment by sharkwouter** [comment]
I don't actually think it's a problem with actkbd. Since it does show me button codes when I run it like "actkbd -n -s -d /dev/input/event26", but  If you're not using it anymore, I guess you should probably remove it. I still think Steam needs a kill switch, though, Steam betas often have some problems which cause input issues.

**Comment by johnv-valve** [comment]
The problem is the udev rules which launch actkbd. systemd does not allow you to use RUN for a long-running daemon and kills actkbd soon after launching it. It needs to be migrated to a real systemd service.

**Comment by ProfessorKaos64** [comment]
Shark, I have had 2 folks email me about the custom rules you set (with the gamepad vs keyboard) not working, but I haven't had time to test. Could it be related to these rule sets not working as you describe?

**Comment by sharkwouter** [comment]
@ProfessorKaos64 Correct, my vaporos-binds-xbox360 package relies uses a udev rule similar to this one to launch actkbd.

**Comment by MattDevo** [comment]
following, since rebooting via controller / vaporos-binds-xbox360 package was my only option now that the reboot menu option is gone.
